src/WebJobs.Script.WebHost/WebJobsServiceCollectionExtensions.cs

```diff
@@ -115,8 +114,10 @@ public static IServiceProvider AddWebJobsScriptHost(this IServiceCollection serv
             // Temporary - This should be replaced with a simple type registration.
             builder.Register<IExtensionsManager>(c =>
             {
-                var hostInstance = c.Resolve<WebScriptHostManager>().Instance;
-                return new ExtensionsManager(hostInstance.ScriptConfig.RootScriptPath, hostInstance.Logger, hostInstance.ScriptConfig.NugetFallBackPath);
```

Comment:
Previously this app level service was binding to host instance values, like nuget fallback path, which comes from reading host.json. However when host.json is changed this ExtensionManager instance wouldn't be reinitialized, right? So I think there were existing problems with this initialization.

Here I'm tying it to non host specific values. Passing null for the nuget fallback path means the default logic kicks in. Is that OK?

Reply:
This registration is transient, so it shouldn't be a problem. Also, changing in `hosting`

Reply:
Transient as, you'll get a new instance from the container every time the service is requested.